pages/customers/fey.mdx

```diff
@@ -0,0 +1,78 @@
+export const title = '50x faster and 50x cheaper. How Fey leverages Inngest in data-intensive processes.'
+export const companyName = 'Fey'
+export const logo = '/assets/customers/fey/fey-icon.svg'
+export const logoScale = 2;
+export const quote = {
+  text: `As soon as companies release their data, we want to have it in Fey. With our old tool it would take up to a day for them to show up. And with Inngest it's almost instant.`,
+  attribution: {
+    name: 'Dennis Brotzky',
+    title: 'CTO, Co-founder',
+  },
+  avatar: '/assets/customers/fey/dennis-brotzky.jpg',
+}
+export const companyURL = 'https://www.feyapp.com'
+export const companyDescription = `Fey is a personal finance app that that's making making financial research more accessible and effortless for anyone.`
+export const ogImage = '/assets/customers/fey/og-image-fey.png'
+
+_Fey is a personal finance app that that's making making financial research more accessible and effortless for anyone. Built by a small team, the app uses Next.js, Inngest, and Render, leveraging serverless functions wherever appropriate._
+
+## The Challenge: Quickly process massive datasets
+
+Fey relies on different financial data providers for data like price history, earnings, analyst estimates, financials, statistics, among many others. At the core of this data-intensive application are data pipelines which fetch, parse, clean, sort, and store the data. Also, different pieces of data come from different providers which adds additional complexity. Most of these complicated pipelines are run on schedules and may contain many discrete steps.
+
+![A screenshot of Fey's dashboard](/assets/customers/fey/fey-dashboard.png)
+
+The team a tool that can quickly orchestrate tasks and reliably trigger scheduled jobsand which would allow them to easily execute these background processes from the client through an API.
```

Comment:
```suggestion
The team a tool that can quickly orchestrate tasks and reliably trigger scheduled jobs and which would allow them to easily execute these background processes from the client through an API.
```